assets/css/sensei-course-theme/layout.scss

```diff
@@ -1,9 +1,10 @@
 .sensei-course-theme {
 
-	--header-height: 75px;
+	--header-height: var(--focus-header-height, 75px);
 	--sidebar-width: 300px;
 	--top-offset: 0px;
 	--content-padding: 32px;
+	--full-header-height: calc(var(--header-height) + var(--top-offset));
```

Comment:
I created this variable, so it's easier to re-use in the places we need to get the header space. I also fixed the `header-height` when focus mode is activated setting a variable there.